website/docs/docs/deploy/deploy-environments.md

```diff
@@ -16,15 +16,18 @@ A dbt Cloud project can have multiple deployment environments, providing you the
 To learn different approaches to managing dbt Cloud environments and recommendations for your organization's unique needs, read [dbt Cloud environment best practices](/guides/set-up-ci).
 ::: 
  
-This page reviews the different types of environments and how to configure your deployment environment in dbt Cloud. 
+Learn more about development vs. deployment environments in [dbt Cloud Environments](/docs/dbt-cloud-environments).
 
-import CloudEnvInfo from '/snippets/_cloud-environments-info.md';
+There are three types of deployment environments that serve different needs:
+- **Production**: Environment for transforming data and building pipelines for production use.
```

Comment:
@matthewshaver What do you think about adding more context and making the definitions more specific (@jtcohen6 might need to keep me honest as I'm not sure these are correct):

Production: Transform data and build pipelines for live usage in production systems. For example, you can transform customer data and generate reports for real-time analytics.

Staging: Test production tools with restricted access to actual production data. For example, you can test new data transformation scripts before deploying them to the production environment.

General:  Develop and deploy in this versatile environment that's not intended for final production workflows. You can experiment with new data models or conduct performance tests before moving them to production.

Reply:
> Production: Transform data and build pipelines for live usage in production systems. For example, you can transform customer data and generate reports for real-time analytics.

It wouldn't be accurate to place emphasis on "live" / "real-time." The key difference is that Production is the only environment that is being referenced by downstream dashboards/reporting, applications, etc. This also contains real data, including (potentially) PII/PHI (personally identifiable information / personal health information)

> Staging: Test production tools with restricted access to actual production data. For example, you can test new data transformation scripts before deploying them to the production environment.

- "Test production tools" -> "Continuously deploy changes and verify reporting."
- Many companies (but not all) will have actually different/restricted data in Production and non-Production environments, where non-Prod = Staging + Development. Or, it's the same data but they leverage data warehouse capabilities to dynamically mask/filter it depending on who's querying.
- "For example, you can test new data transformation scripts before deploying them to the production environment." -> This is conceptually right, but I'd be inclined to say "new and changed dbt models" rather than "new data transformation scripts"

> General: Develop and deploy in this versatile environment that's not intended for final production workflows. You can experiment with new data models or conduct performance tests before moving them to production.

We are renaming "General" to "Other." This is just a catch-all for needing to have additional custom environments. There is nothing special about these, and they do not carry the same functionality or canonical meaning as "Production" or "Staging."